client/src/App.js

```diff
@@ -223,6 +226,13 @@ export default class App extends Component {
     }
   };
 
+  handleFromNumberSelectChange = e => {
+    console.log(e.option);
```

Comment:
Might be worth adding a string to the front of this so you know what's being logged.

Reply:
Ended up removing this. This was a leftover from figuring out how to modify the select component.